layouts/partials/cards/job_card/job_card.html

```diff
@@ -0,0 +1,56 @@
+
+{{ $path := printf "%s%s%s%s%s" "/content/jobs" "/" (lower .role) "/" (urlize (lower .jobData.name)) }}
+{{ $indexPath := printf "%s%s" $path "/_index.md" }}
+{{ $advancedGuide := printf "%s%s" $path "/advanced-guide.md" }}
+{{ $basicGuide := printf "%s%s" $path "/basic-guide.md" }}
+{{ $bis := printf "%s%s" $path "/best-in-slot.md" }}
+{{ $faq := printf "%s%s" $path "/faq.md" }}
+{{ $jobChanges := printf "%s%s" $path "/job-changes.md" }}
+{{ $levelingGuide := printf "%s%s" $path "/leveling-guide.md" }}
+{{ $openers := printf "%s%s" $path "/openers.md" }}
+{{ $skillsOverview := printf "%s%s" $path "/skills-overview.md" }}
+{{ $statPriority := printf "%s%s" $path "/stat-priority.md" }}
+
+{{ $pages := slice
+  (.siteVar.GetPage $levelingGuide)
+  (.siteVar.GetPage $basicGuide)
+  (.siteVar.GetPage $advancedGuide)
+  (.siteVar.GetPage $skillsOverview)
+  (.siteVar.GetPage $openers)
+  (.siteVar.GetPage $faq)
+  (.siteVar.GetPage $bis)
+  (.siteVar.GetPage $jobChanges)
+  (.siteVar.GetPage $statPriority)
+}}
```

Comment:
I tried to run a when loop to do these GetPage, but for some reason it returns an error. Sorry for the duplicates

Reply:
Ohhhh you have to use `( )` like that. Interesting

Reply:
Ok I found a nice way to do it